common/test/acceptance/pages/studio/users.py

```diff
@@ -107,18 +107,39 @@ def modal_dialog_text(self, dialog_type):
         """ Gets modal dialog text """
         return self.q(css='.prompt.{dialog_type} .message'.format(dialog_type=dialog_type)).text[0]
 
+    def wait_until_no_loading_indicator(self):
+        """
+        When the page first loads, there is a loading indicator and most
+        functionality is not yet available. This waits for that loading to finish
+        and be removed from the DOM.
+
+        This method is different from wait_until_ready because the loading element
+        is removed from the DOM, rather than hidden.
+
+        It also disables animations for improved test reliability.
+        """
+
+        self.wait_for(
+            lambda: not self.q(css='.ui-loading').present,
+            "Wait for page to complete its initial loading"
+        )
+        disable_animations(self)
```

Comment:
That's a fair question, but I don't know the answer. I only split out the existing method into two.